### ISSUE TITLE
Could you please help me with a custom loss function?

### ISSUE THREAD

**Issue by hima111997** [Apress/artificial-neural-networks-with-tensorflow-2#9]
Hello I was trying to do multi classification on CIFAR-10 data set and I wanted to write a custom loss function that was published in this paper: https://www.hindawi.com/journals/cin/2021/6660961/#abstract
my implementation of the loss function is in the colab link but when I started training the loss did not change from 34.1734
can you help me, please?
Thanks
Colab link: https://colab.research.google.com/drive/1Mqn6gIk5voX-9mNCjUDr8I2vGyLd8r87?usp=sharing

**Comment by profsarang** [comment]
May I suggest that you take help from the author of this article.

**Comment by profsarang** [comment]
How to write a custom loss function is covered in basic ML course.
